orm/uint64_index.go

```diff
@@ -40,6 +44,11 @@ func (i UInt64Index) Get(ctx HasKVStore, searchKey uint64) (Iterator, error) {
 	return i.multiKeyIndex.Get(ctx, EncodeSequence(searchKey))
 }
 
+// GetPaginated returns a result iterator for the searchKey based on pageRequest.
+func (i UInt64Index) GetPaginated(ctx HasKVStore, searchKey uint64, pageRequest *query.PageRequest) (Iterator, error) {
+	return i.multiKeyIndex.GetPaginated(ctx, EncodeSequence(searchKey), pageRequest)
+}
```

Comment:
Should we add a test for this function?

Reply:
It was already tested through `TestPaginate` in `orm/iterator_test.go` but I have also added some additional tests in `orm/uint64_index_test.go` as well as `orm/index_test.go`.